File: _persist/temp/venda/CSharp/Regraimposto.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("REGRAIMPOSTO")]
    public class Regraimposto : CollectionItem
    {
        [Campo("ID_REGRAFISCAL", CampoTipo.tfKey)]
        public int Id_Regrafiscal { get; set; }
        [Campo("CD_IMPOSTO", CampoTipo.tfKey)]
        public int Cd_Imposto { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.tfReq)]
        public double Pr_Aliquota { get; set; }
        [Campo("PR_BASECALCULO", CampoTipo.tfReq)]
        public double Pr_Basecalculo { get; set; }
        [Campo("CD_CST", CampoTipo.tfReq)]
        public string Cd_Cst { get; set; }
        [Campo("CD_CSOSN", CampoTipo.tfNul)]
        public string Cd_Csosn { get; set; }
        [Campo("IN_ISENTO", CampoTipo.tfReq)]
        public string In_Isento { get; set; }
        [Campo("IN_OUTRO", CampoTipo.tfReq)]
        public string In_Outro { get; set; }
    }
}

File: _persist/temp/venda/Java/Regraimposto.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("REGRAIMPOSTO")
public class Regraimposto extends CollectionItem
{
    @Campo("ID_REGRAFISCAL", CampoTipo.tfKey)
    public Integer Id_Regrafiscal;
    @Campo("CD_IMPOSTO", CampoTipo.tfKey)
    public Integer Cd_Imposto;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("PR_ALIQUOTA", CampoTipo.tfReq)
    public Float Pr_Aliquota;
    @Campo("PR_BASECALCULO", CampoTipo.tfReq)
    public Float Pr_Basecalculo;
    @Campo("CD_CST", CampoTipo.tfReq)
    public String Cd_Cst;
    @Campo("CD_CSOSN", CampoTipo.tfNul)
    public String Cd_Csosn;
    @Campo("IN_ISENTO", CampoTipo.tfReq)
    public String In_Isento;
    @Campo("IN_OUTRO", CampoTipo.tfReq)
    public String In_Outro;
}